### PR TITLE
Package id is not affected by version ranges

### PR DESCRIPTION
**Pull request by prince-chrismc**
Some good questions came up in https://github.com/conan-io/conan-center-index/pull/8831 and I could not find the answer in the docs. Since I dug in the GitHub issues and found my answer figured it was worth add to the docs

I picked the two obvious spot where I first looked

As always feedback is welcomed 🙏

### CODE REVIEW

File: creating_packages/define_abi_compatibility.rst [hunk 1]
```diff
@@ -630,6 +630,9 @@ All the modes can be applied to all dependencies, or to individual ones:
   marked as such. These dependency graphs cannot be built in a single invocation, because they are intended
   for CI systems, in which a package creation/built is called for each package in the graph.
 
+.. note::
+  
+  Version ranges are not used calculate the `package_id` only the resolved version in the graph is used
```
Comment (memsharded) [inline creating_packages/define_abi_compatibility.rst:635]:
```suggestion
  
  Version ranges are not used calculate the ``package_id`` only the resolved version in the graph is used
```

File: versioning/version_ranges.rst [hunk 2]
```diff
@@ -66,3 +66,7 @@ The order of search for matching versions is as follows:
 - If the :command:`--update` parameter is used, then the existing packages in the local conan cache will not be used, and the same search of the
   previous steps is carried out in the remotes. If new matching versions are found, they will be retrieved, so subsequent calls to
   :command:`install` will find them locally and use them.
+
+.. note::
+
+   Version ranges are not used in generating `package_id`s those are always determined by the resolved graph.
```
Comment (memsharded) [inline versioning/version_ranges.rst:72]:
```suggestion

   Version ranges are not used in generating ``package_id``s those are always determined by the resolved graph.
```

**Review by memsharded — APPROVED** [review]
Good observation, totally right that the range is not used, as the range does not affect the binary, only the version used can affect the binary.

**Review by uilianries — APPROVED** [review]

File: versioning/version_ranges.rst [hunk 3]
```diff
@@ -66,3 +66,7 @@ The order of search for matching versions is as follows:
 - If the :command:`--update` parameter is used, then the existing packages in the local conan cache will not be used, and the same search of the
   previous steps is carried out in the remotes. If new matching versions are found, they will be retrieved, so subsequent calls to
   :command:`install` will find them locally and use them.
+
+.. note::
+
+   Version ranges are not used in generating ``package_id``s those are always determined by the resolved graph.
```
Comment (memsharded) [inline versioning/version_ranges.rst:72]:
```suggestion
   Version ranges are not used in generating ``package_id`` those are always determined by the resolved graph.
```

**Pull request merged**